code/computational_geometry/src/bresenham_line/Bresenham_Line_Algorithm.py

```diff
@@ -0,0 +1,72 @@
+# Import graphics library
+from graphics import *
+
+import time
+
+
+def Bresenham_Line(xa, ya, xb, yb):
+
```

Comment:
Format code using `black` python formatter. Remove this newline. Also call the function name `bresenham_line`.